docs/src/app/components/pages/components/buttons.jsx

```diff
@@ -74,25 +74,25 @@ export default class ButtonPage extends React.Component {
           },
           {
             name: 'labelPosition',
-            type: 'oneOf ["before", "after"]',
+            type: 'oneOf [before, after]',
             header: 'default: "before"',
             desc: 'Place label before or after the passed children',
           },
           {
             name: 'linkButton',
-            type: 'bool',
+            type: 'boolean',
```

Comment:
If we're sticking to React PropTypes, `boolean`s would need to change back to `bool`s